requirements/default.txt

```diff
@@ -246,9 +250,12 @@ importlib==1.0.3 \
     --hash=sha256:65f342a604a2e1028707c5e055266ab2431c26e20fe10780b423320870884dac
 
 # Improved template engine
-Jinja2==2.8 \
-    --hash=sha256:1cc03ef32b64be19e0a5b54578dd790906a34943fe9102cfdae0d4495bd536b4 \
-    --hash=sha256:bc1ff2ff88dbfacefde4ddde471d1417d3b304e8df103a7a9437d47269201bf4
+# Code: https://github.com/pallets/jinja
+# Changes: http://jinja.pocoo.org/docs/2.10/changelog/
```

Comment:
Several changes that will impact Kuma
- urlize now sets “rel noopener” by default.
- with and autoescape tags are built-in
- Added a (default on?) trimmed modifier to ``{% trans %}``